snowflake-api/Cargo.toml

```diff
@@ -13,6 +13,7 @@ version = "0.6.0"
 
 [features]
 cert-auth = ["dep:snowflake-jwt"]
+file = ["dep:glob", "dep:tokio", "dep:object_store"]
```

Comment:
I think it's a bit self-defeating to hide all features by-default. The smallest library is the one which does nothing, but also the most useless one. Enabling reasonable amount of features and avoiding user surprise (getting an unsupported error in runtime) is a better way to go.

Reply:
As for making tokio optional, `reqwest` needs it either way